README.md

```diff
@@ -1,34 +1,92 @@
-insalan.fr
-==========
+# insalan.fr
 
 [![Build Status](https://travis-ci.org/InsaLan/insalan.fr.svg?branch=master)](https://travis-ci.org/InsaLan/insalan.fr)
 [![Maintainability](https://api.codeclimate.com/v1/badges/68707ca6cd1a2b332dc4/maintainability)](https://codeclimate.com/github/InsaLan/insalan.fr/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/68707ca6cd1a2b332dc4/test_coverage)](https://codeclimate.com/github/InsaLan/insalan.fr/test_coverage)
 
 Website to handle esport tournament
 
-Installation
-------------
 
-Download vendors
-Considering you have composer.phar installed and in your PATH :
+## Installation
+
+The project need the following package and php-extension :
+
+Package | Comment
+------- | -------
+git |
+php | php7.0
+composer |
+mariadb-server | if use with local database
+zip | not necessary (only for unzip during install)
+
+PHP extension | Comment
+------------- | -------
+php-curl |
+php-intl | date translation
+php-mbstring | doctrine:create:schema:create
+php-mysql |
+php-xml |
+
+
+### Linux
+
+
+#### Base packages
+
+Update your local packages
+
+```bash
+sudo apt-get update && apt-get upgrade
+```
+
+Install git and clone the repository
 
 ```bash
+sudo apt-get install git
+git clone https://github.com/InsaLan/insalan.fr
+```
+
+It is recommended to get `composer.phar` in your PATH : https://getcomposer.org/doc/00-intro.md
+
+Install the remaining packages
+
+```bash
+sudo apt-get install php php-curl php-intl php-mbstring php-mysql php-xml mariadb-server zip
+```
+
+Proceed with the install and accept default settings
+
+```bash
+cd insalan.fr
 composer.phar install
 ```
 
-If not set with composer.phar, configure symfony2 : 
+At this point, you should be able to run the web server but any pages you try to access will return an error since the database is not setup.
+
+
+#### mariadb
+
+Log in mariadb as root
 
 ```bash
-cp app/config/parameters.yml.dist app/config/parameters.yml
-vim app/config/parameters.yml
+sudo mysql -u root -p
+```
+
+Create database, user and grant access
+
+```mariadb
+CREATE DATABASE insalan;
```

Comment:
The script was not able to connect to mariaDB since there was not a user 'insalan' in the database. Maybe because I was not logging in as 'insalan' on my test system.

I could try this week end or the next one (exams next week...)